frontend/src/components/Login.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+    let active = false;
+
+    function isActive() {
+        return active ? "is-active" : "";
+    }
+</script>
+
+<style>
+    .modal-content {
+        width: 31.25em;
+        height: 35.5em;
+        background: #D9D9D9;
+        border-radius: 7px;
+    }
+
+    .link:hover {
+        text-decoration: underline;
+        color: #276BB0;
+    }
+
+    #login-contents {
+        margin: 2.5em 3.125em 0em;
+    }
+</style>
+
+<button class="button is-light" on:click={() => { active = true; }}>
+    <strong>Log in</strong>
+</button>
+
+{#if active}
+    <div class="modal {isActive()}">
+    <div class="modal-background" color=""/>
+    <div class="modal-content">
+        <div class="is-flex is-flex-direction-column">
+            <div class="is-align-self-flex-end">
+                <button class="delete is-large" style="background: #D9D9D9" aria-label="close" on:click={() => { active = false; }}></button>
+            </div>
+            <div class="is-flex is-flex-direction-column pt-4" id="login-contents">
+                <p class="title is-1 has-text-black">Log in</p>
+
+                <div class="field mt-5">
+                    <div class="control">
+                        <input class="input" type="email" placeholder="    Email" />
+                    </div>
+                </div>
+                <div class="field mt-3">
+                    <div class="control">
+                        <input class="input" type="password" placeholder="    Password" />
+                    </div>
+                </div>
+
+                <div>
+                    <a class="link" href="/">Forgot your password?</a>
+                </div>
+
+                <div class="is-align-self-flex-end pt-5">
+                    <div class="field">
+                        <div class="control">
+                            <button class="button is-link is-medium"><p class="px-3">Log in</p></button> 
```

Comment:
Stylistic suggestion:
`<button class="button is-link is-medium"><p class="px-3"><strong>Log in</strong></p></button>`